src/operator/tensor/elemwise_binary_broadcast_op_extended.cc

```diff
@@ -33,7 +44,33 @@ NNVM_REGISTER_OP(_backward_broadcast_power)
                                                               mshadow_op::power_rgrad>);
 
 MXNET_OPERATOR_REGISTER_BINARY_BROADCAST(broadcast_maximum)
-.describe(R"code(Element-wise maximum of array elements with broadcasting.
+.describe(R"code(Returns element-wise maximum of the input arrays with broadcasting.
+
+It compares input arrays and returns the new array having the element-wise maxima.
```

Comment:
Avoid pronouns where the subject is ambiguous: what does "it" mean? Also the function returns "a new array" not "the new array".

Change to:
"This function compares two input arrays and returns a new array whose shape is an element-wise max of the input array shapes."

Is this correct?